### ISSUE TITLE
Tutorial: SWESARR

### ISSUE THREAD

**Issue by scottyhq** [uwhackweek/schedule-2024#16]
**Lead:** Dylan Boyd, Elodie Macorps
**Date:** 21/08/2024
**Start Time:** 0900
**Duration:** 45
**Description:** All about SWESARR

<details><summary>Details</summary>
<p>

### Learning Outcomes

* Show people how radar is different from optical
* Visualize SWESARR data
* showcase an inversion algorithm with snow structures

### Technical Needs (GPUs? Large file storage? Unique libraries?)

</p>
</details>

**Comment by drboyd** [comment]
Hi @scottyhq . I'm hoping to add @emacorps to the Hackweek team to help work on this tutorial. Could you please invite her to the uwhackweek group?

**Comment by scottyhq** [comment]
Yep - looks like there is a pending application, so the should be an invite email from GitHub to join the organization
